mentors/typescript/i.json

```diff
@@ -1 +1,8 @@
-[]
+[{
+  "github_username": "Inkdpixels",
+  "name": "Tyll Weiß",
+  "link_text": null,
+  "link_url": null,
+  "avatar_url": null,
+  "bio": "I love TypeScript for for it's advance in features and type-safety, all while still being close to Javascript. I've written and maintained quite a bit of TypeScript projects, both personally and professionally."
+}]
```

Comment:
Like the other json file same formatting change.
```json
[
  {
    "github_username": "Inkdpixels",
    "name": "Tyll Weiß",
    "link_text": null,
    "link_url": null,
    "avatar_url": null,
    "bio": "I love TypeScript for for it's advance in features and type-safety, all while still being close to Javascript. I've written and maintained quite a bit of TypeScript projects, both personally and professionally."
  }
]
```

Reply:
Done :)